src/template.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>todo</title>
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+<p>something</p>
+</body>
+
+</html>
```

Comment:
- [ ] In package.json you stated `index.js` as an a main entry. However, you are missing the index.js file in your `src` folder. I kindly suggest to create index.js file and use Use ES6 modules, with import and export as suggested in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md#project-requirements).
- [ ] In addition, Kindly follow the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), to write plain HTML markup with minimum styling (just to make the layout),

Your design 😎 
![image](https://github.com/DjibrilM/Leaderboard/assets/99394374/49a0f9e7-1cfe-4c0e-89a7-c2f9a2b443ec)